src/global.css

```diff
@@ -1,32 +1,114 @@
-body{
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
-  margin:auto;
-  background-color: #262626c9;
-  width: 55%;
-}
-.text{
-  background-color: #121212;
-  border: 0.2rem solid black;
-  color: #FFC529;
-  padding: 1rem 1rem;
-  margin-top: 5rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
-  box-shadow: rgba(0, 0, 0, 0.25) 0 3.4rem 3.5rem, rgba(0, 0, 0, 0.12) 0 -0.75rem 1.8rem, rgba(0, 0, 0, 0.12) 0 1rem 1.4rem, rgba(0, 0, 0, 0.17) 0 2rem 1.8rem, rgba(0, 0, 0, 0.09) 0 -0.8rem 1rem;
-}
-.socials > a{
-  font-size: 1.8rem;
-  padding-left: 1rem;
-  color:#FFC529;
-}
-.socials >a:hover{
-  color:#af8206;
-  cursor: pointer;
-  transition: all 0.3s smooth;
+@import url("https://fonts.googleapis.com/css2?family=Inder&display=swap");
+
+:root {
+	--primaryYellow: #ffc529;
+	--secondayYellow: #feb74c;
+	--white: #ffffff;
+	--black: #000000;
+	--mainBackground: #232323;
+}
+
+body {
+	background-color: var(--mainBackground);
+	font-family: "Inder", "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+		"Lucida Sans", Arial, sans-serif;
+	padding: 1rem;
+	font-size: 16px;
+}
+
+.navbar {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	align-items: center;
+	gap: 10px;
+	color: var(--primaryYellow);
+}
+.navbar img {
+	width: 60px;
+	height: 33px;
+}
+.navbar h2 {
+	font-style: normal;
+	font-weight: 400;
+	font-size: 40px;
+	line-height: 50px;
+	display: flex;
+	align-items: center;
+	text-align: center;
+	margin: 0;
+}
+
+/**************
+**** MAIN ****
+**************/
+.main {
+	display: grid;
+	align-items: center;
+	grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));
+}
+
+.main-photos {
+	text-align: center;
+}
+
+/**************
+** MAIN TEXT **
+**************/
+.main-text h1 {
+	font-weight: 400;
+	font-size: 7rem;
+	line-height: 7rem;
+	color: var(--primaryYellow);
+	text-shadow: 0px 4px 20px #000000;
+	margin-top: 3.125rem;
+	margin-bottom: 1.563rem;
+	display: inline;
+}
+@media screen and (max-width: 900px) {
+	.main-text h1 {
+		font-size: 5rem;
+		text-align: center;
+	}
+}
+
+.main-text p {
+	font-size: 1rem;
+	line-height: 1.3rem;
+	color: var(--white);
+}
+.main-text button {
+	background-color: var(--primaryYellow);
+	border-radius: 2.5rem;
+	font-weight: 700;
+	font-size: 1.2rem;
+	line-height: 29px;
+	width: max-content;
+	padding: 0.313rem 1.25rem;
+	margin: 0.5rem 0;
+	color: var(--mainBackground);
+}
+button:hover {
+	background-color: var(--black);
+	color: var(--secondayYellow);
+}
+
+/**************
+* MAIN PHOTOS *
+**************/
+.main-photos h3 {
```

Comment:
I feel specificity make causes an issue here, could you use a karat to ensure that the targeted element is going to be affected?

Example .main-photos > h3. (This will ensure that the h3 in the .main-photos class will be affected and not any other h3.)

Reply:
Added the karats